packages/graphile-build-pg/package.json

```diff
@@ -3,10 +3,11 @@
   "version": "4.0.0-rc.5",
   "description": "Build a GraphQL schema by reflection over a PostgreSQL schema. Easy to customize since it's built with plugins on graphile-build",
   "main": "node8plus/index.js",
+  "types": "node8plus/index.d.ts",
   "scripts": {
     "test": "jest",
-    "prepack": "mkdir -p node8plus && babel -s true --out-dir node8plus src && flow-copy-source src node8plus",
-    "watch": "mkdir -p node8plus && babel -s true --watch --out-dir node8plus src"
+    "prepack": "mkdir -p node8plus && babel --copy-files -s true --out-dir node8plus src && flow-copy-source src node8plus",
+    "watch": "mkdir -p node8plus && babel --copy-files -s true --watch --out-dir node8plus src"
```

Comment:
`--copy-files` is needed to copy the `.d.ts` files.

Reply:
Graphile-build-pg is still in JS+Flow, so I’ve just added manual  .d.ts files for now.